packages/gatsby-plugin-mdx/utils/gray-matter-wrapper/index.js

```diff
@@ -0,0 +1,8 @@
+const grayMatter = require(`gray-matter`)
+
+module.exports = (input, options) => {
+  const matter = grayMatter(input, options.grayMatter || {})
+  const { data: origData, excerpt, content } = matter
+  const data = excerpt === `` ? origData : { excerpt, ...origData }
```

Comment:
Can `excerpt` not be `null` or `undefined` here in case of absence? Just wondering whether a `!excerpt` wouldn't be better here, if it can also be the empty string.